doc/engsys_resource_management.md

```diff
@@ -0,0 +1,94 @@
+## Resource management guidelines
+
+This document contains guidelines for creating and managing Azure resources in the Azure SDK Engineering System
+subscriptions. EngSys has automation that will delete resources based on the criteria outlined below. This is to
+improve our overall security and keep us in compliance with company policies.
+
+  * [Managing Dev Resources in the Playground Subscription](#managing-dev-resources-in-the-playground-subscription)
+     * [Resource Groups](#resource-groups)
+     * [Role Assignments](#role-assignments)
+  * [Managing CI resources in the test subscription](#managing-ci-resources-in-the-test-subscription)
+     * [Resource Groups](#resource-groups-1)
+     * [Role Assignments](#role-assignments-1)
+
+
+## Managing Dev Resources in the Playground Subscription
+
+This section applies to resource groups located in the `Azure SDK Developer Playground` subscription.
+
+Currently the automation will inspect resource groups and role assignments only.
+
+### Resource Groups
+
+The automation will delete all resource groups in the playground subscription DAILY, unless they meet one of the following criteria:
+
+- The resource group starts with a valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`, followed by an
+  optional hyphen and extra characters.
+    - Valid group name examples: `myalias`, `myalias-test-foobar` for `myalias@microsoft.com`
+- The resource group contains a tag with the name `owners` where the value is a csv formatted string that contains at
+  least one valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`. This convention should only be used
+  when it is not possible to name the resource group with an alias (for example, inner groups auto-created by a resource
+  provider like AKS).
+    - Valid owner tag examples: `owners: myalias`, `owners: myalias,anotheralias,lastalias`
+- The resource group contains a tag with the name `DeleteAfter` and an ISO8601 formatted date value, where the date is
+  not greater than 7 days in the future.
+    - If the `DeleteAfter` value is in the past, or greater than 7 days in the future, it will be deleted.
+    - If you have a resource group for which you would like to extend the lifetime, update the `DeleteAfter` tag to a
+      future date to renew the lease.
+    - Valid date tag format: `DeleteAfter: 2022-01-29T00:35:48.9372617Z`
+      Example extending resource group lease by three days:
+      ```
+      # powershell
+      Set-AzResourceGroup -Name <group name> -Tag @{ DeleteAfter = [DateTime]::UtcNow.AddDays(3).ToString("o") }
```

Comment:
This works in general, but do we want to mention the *eng/common/TestResources/Update-TestResources.ps1* script as well? For TestResources-deployed resources, this would be the easiest way.

Reply:
Good point, I'll add a mention for that.